WTWSMS/decisions/realm_decisions.txt

```diff
@@ -662,7 +662,14 @@ decisions = {
 			has_global_flag = viking_age_started
 			NOT = { has_global_flag = varangian_guard_founded }
 			any_playable_ruler = {
-				religion = norse_pagan
+				OR = {
+					religion = norse_pagan
+					religion = germanic_pagan
+					religion = norse_pagan_reformed
+					religion = germanic_pagan_reformed
+					religion = saxon_pagan
+					religion = saxon_pagan_reformed
+				}
```

Comment:
It's not a bad idea, because the Saxon Pagan religion is in the code and not subject to removal. It was never properly implemented, but it has always intended to be implemented. @MaelgwnGwynedd outlined a whole book of reasons why Saxon/Wodenic Paganism should be its own branch unique to the Jutes, Angles, and Saxons. All we have to do is change the provincial religion and the religion of the starting characters, and we can begin implementing Maelgwynn's flavor.